RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -350,33 +350,55 @@ class RNUnifiedContacts: NSObject {
     return emailAddresses
   }
   
-  func transformNumber(number: NSDictionary) -> CNLabeledValue {
+  func convertPhoneNumberToNSDictionary(phoneNumber: NSDictionary) -> CNLabeledValue {
     var label = String()
-    if (number["label"] as! String == "home") {
-      label = CNLabelHome
-    } else if (number["label"] as! String == "work") {
-      label = CNLabelWork
-    } else if (number["label"] as! String == "mobile") {
-      label = CNLabelPhoneNumberMobile
+    switch (phoneNumber["label"] as! String) {
+      case "home":
+        label = CNLabelHome
+      case "work":
+        label = CNLabelWork
+      case "mobile":
+        label = CNLabelPhoneNumberMobile
+      case "iPhone":
+        label = CNLabelPhoneNumberiPhone
+      case "main":
+        label = CNLabelPhoneNumberMain
+      case "home fax":
+        label = CNLabelPhoneNumberHomeFax
+      case "work fax":
+        label = CNLabelPhoneNumberWorkFax
+      case "pager":
+        label = CNLabelPhoneNumberPager
+      case "other":
+        label = CNLabelOther
+      default:
```

Comment:
What if someone uses a different label than what we have here? By this code it'll just set it to blank `""`.

Do we want to handle custom labels? I know iOS Contacts allows you to create your own labels. 

Thoughts?